modules/va_notify/app/sidekiq/va_notify/email_job.rb

```diff
@@ -17,8 +17,13 @@ class EmailJob
       StatsD.increment("sidekiq.jobs.#{job_class.underscore}.retries_exhausted")
     end
 
-    def perform(email, template_id, personalisation = nil, api_key = Settings.vanotify.services.va_gov.api_key)
-      notify_client = VaNotify::Service.new(api_key)
+    def perform(email, template_id, personalisation = nil, api_key = Settings.vanotify.services.va_gov.api_key,
+                callback_options = nil)
+      notify_client = if callback_options
+                        VaNotify::Service.new(api_key, callback_options)
+                      else
+                        VaNotify::Service.new(api_key)
+                      end
```

Comment:
I think we don't need this conditional block since Service has callback_options as optional with a default value of nil.